shell/platform/windows/platform_handler.h

```diff
@@ -22,10 +22,14 @@ namespace flutter {
 class FlutterWindowsEngine;
 class ScopedClipboardInterface;
 
+enum ExitType {
```

Comment:
```suggestion
enum class ExitType {
```
In order to avoid the implicit type conversion issues present in traditional enums.